scipy/signal/tests/test_windows.py

```diff
@@ -303,6 +304,23 @@ def test_basic(self):
                         [0, 0.25, 0.75, 1.0, 0.75, 0.25, 0])
 
 
+class TestLanczos(object):
+
+    def test_basic(self):
+        assert_allclose(signal.lanczos(6, False),
```

Comment:
Can you add a small comment about where these hard-coded values come from (hand calculations, Octave, etc.)?